app/views/patients/_patient_dashboard.html.erb

```diff
@@ -1,3 +1,12 @@
+<div class="row">
+  <div class="col-sm-4">
+    <% if current_user.admin? %>
+      <% if not patient.pledge_sent %>
+        <%= link_to delete_patient_button, delete_patient_path(patient),  class:'delete-patient-button', method: :delete, data: {confirm: "Are you sure you want to delete #{patient.name}?"} %>
```

Comment:
Might want to check out `button_to`, if memory serves -- might do both the `link_to` and `content_tag` in one.

Since we want to take advantage of the REST route we get from `resources`, go ahead and use `patient_path(patient)` rather than `delete_patient_path(patient)`.